aries-site/src/components/feedback/Question.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormField,
+  TextArea,
+  Text,
+  RadioButtonGroup,
+  FileInput,
+} from 'grommet';
+import { StarRating } from './StarRating';
+import { ThumbsRating } from './ThumbsRating';
+
+export const Question = ({ formProps, label, inputProps, kind, name }) => {
+  const accessibilityProps = {
+    id: name,
+    name,
+  };
+
+  const formats = {
+    star: { render: StarRating, border: false },
+    thumbs: {
+      render: ThumbsRating,
```

Comment:
It's a little weird how after I click the focus indicator is visible around the entire formfield (which previously didn't have any border)

<img width="383" alt="Screen Shot 2022-09-13 at 1 11 20 PM" src="https://user-images.githubusercontent.com/12522275/189999861-ca22154e-982d-42b0-878b-476124909d95.png">

Reply:
This is still a strange behavior, but I'm fine handling it as a separate PR as it's more related to FormField and custom content.